specs/triggers/triggerbinding.spec

```diff
@@ -0,0 +1,70 @@
+# Verify triggerbindings spec
+
+Pre condition:
+  * Validate Operator should be installed
+
+## Verify CEL marshaljson function Test: PIPELINES-06-TC01
+Tags: e2e,triggers,non-admin
+Component: Triggers
+Level: Integration
+Type: Functional
+Importance: Critical
```

Comment:
Are we planning to run this as part of acceptance tests?

Reply:
yes